### ISSUE TITLE
Could you please suggest better option for using azure blob storage

### ISSUE THREAD

**Issue by seemamathew** [Azure/azure-storage-cpplite#112]
Hi 

I have to use azure blob storage(c++) to download files from cloud.
While searching I could find following options
- azure-storage-cpplite
- azure-storage-cpp
- azure-storage-blobs
I am using imx6ul board.
I have to download ~1GB.
Could you please suggest which is the better option to me.

Regards
Seema

**Comment by Jinming-Hu** [comment]
> I am using imx6ul board.

Is this an ARM chip?

**Comment by seemamathew** [comment]
> Is this an ARM chip?

Yes it is ARM chip

**Comment by seemamathew** [comment]
Hi,

It will be very helpful if we can get suggestion with in this week.

Thanks in Advance
Seema

**Comment by Jinming-Hu** [comment]
Hi @seemamathew , neither azure-storage-cpplite nor azure-storage-cpp officially supports ARM. But I do know someone ever tried to run azure-storage-cpplite on ARM. But both of them are going to be deprecated in favor of our new [Track2 C++ SDK](https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage).

Although the new Track 2 SDK hasn't officially supported ARM yet, but that's one of our backlog items. Actually I don't think anybody ever tried to build this package on ARM. You can try it, maybe it's already able to work on ARM.

Can you share a link of `azure-storage-blobs`? I'm not quite sure which package you're referring to.

**Comment by seemamathew** [comment]
Hi @Jinming-Hu 

Thank you very much for update.
We will try with Track2 C++ SDK.

Also I found azure-storage-blobs in below path
https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage/azure-storage-blobs

Thanks and Regards
Seema

**Comment by Jinming-Hu** [comment]
> Also I found azure-storage-blobs in below path
> https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage/azure-storage-blobs

Yeah, this is our Track2 SDK.

**Comment by seemamathew** [comment]
> > Also I found azure-storage-blobs in below path
> > https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage/azure-storage-blobs
> 
> Yeah, this is our Track2 SDK.

Thank you for confirmation

**Comment by HimaPraveen** [comment]
Hi @Jinming-Hu,

Whether the new Track2 SDk support multi part download and whether it provides resume support while handling large data.

Thanks in Advance
Hima

**Comment by Jinming-Hu** [comment]
@HimaPraveen Yes, you can specify chunk size and parallelism when downloading a large blob. For each individual HTTP request, if connection failure occurs, the sdk will automatically send a new request to download the remaining data from the offset where connection's lost.
